metadata/mozorg/mission.json

```diff
@@ -90,6 +90,8 @@
     "vi",
     "xh",
     "zh-CN",
-    "zh-TW"
+    "zh-TW",
```

Comment:
Doh! These should be sorted into the list. I filed a quick PR to fix this in the script: https://github.com/mozilla/bedrock/pull/8530